README.md

```diff
@@ -88,11 +88,12 @@ For recording, the executable needs to be compiled with the `-pg`
 (or `-finstrument-functions`) option which generates profiling code
 (calling mcount or __cyg_profile_func_enter/exit) for each function.
 
-Note that, there's an experimental support for dynamic tracing on x86_64
-which doesn't require such (re-)compilations.  Also recent compilers have
-some options to help uftrace to reduce tracing overhead with similar way
-(although it still needs recompilation of your program).  Please see
-[doc/uftrace-record.md](doc/uftrace-record.md) file.
+Note that, there's an experimental support for dynamic tracing on
+x86_64 and AArch64(ARM64) which do not require such (re-)compilations.
```

Comment:
I'm sorry for asking you again. I was wrong. "which doesn't require" is actually correct.
Everything looks good except for it. Thanks.

Reply:
> I'm sorry for asking you again. I was wrong. "which doesn't require" is actually correct.
> Everything looks good except for it. Thanks.

It's okay :) I'll fix it right away!